js/apps/system/_admin/aardvark/APP/frontend/js/views/queryView.js

```diff
@@ -1566,11 +1597,57 @@
 
       // escape all columns but the third (which contains HTML)
       this.myQueriesTableDesc.unescaped = [ false, true, true ];
-      this.myQueriesTableDesc.rows = this.filterRows(this.myQueriesTableDesc.rows);
 
       this.$(this.myQueriesId).html(this.table.render({content: this.myQueriesTableDesc}));
     },
-
+    sortRows: function(rows) { 
+      function compare (a, b) {
+        var x;
+        if (a.name < b.name) {
+          x = -1;
+        } else if (a.name > b.name) {
+          x = 1;
+        } else {
+          x = 0;
+        }
+        return x;
+      }
+      function sortByModifiedAt (a, b) {
+        var x;
+        if (a.modified_at > b.modified_at) {
+          x = -1;
+        } else if (a.modified_at < b.modified_at) {
```

Comment:
Do we need to check here if `modified_at` is available? Will it cause issues if it is not available here? e.g. produce false positives in case of `undefined`?

Reply:
True, good catch! I will add an explicit check, as undefined behaviour can be unpredictable